front/lib/plans/subscription.ts

```diff
@@ -43,6 +43,10 @@ export const internalSubscribeWorkspaceToFreeTestPlan = async ({
     code: freeTestPlan.code,
     name: freeTestPlan.name,
     status: "active",
+    subscriptionId: "no_subscription_id",
```

Comment:
This is a bit disatisfying should we allow null. Feels like it will come to bite us otherwise?